webpack.config.js

```diff
@@ -434,34 +425,71 @@ if (FORCE_WEBPACK_DEV_SERVER || (HAS_WEBPACK_DEV_SERVER_CONFIG && !NO_DEV_SERVER
     watchOptions: {
       ignored: ['node_modules'],
     },
-    publicPath: '/_webpack',
-    proxy: {'!/_webpack': backendAddress},
-    before: app =>
-      app.use((req, _res, next) => {
-        req.url = req.url.replace(/^\/_static\/[^\/]+\/sentry\/dist/, '/_webpack');
-        next();
-      }),
   };
 
-  // XXX(epurkhiser): Sentry (development) can be run in an experimental
-  // pure-SPA mode, where ONLY /api* requests are proxied directly to the API
-  // backend, otherwise ALL requests are rewritten to a development index.html.
-  // Thus completely separating the frontend from serving any pages through the
-  // backend.
-  //
-  // THIS IS EXPERIMENTAL. Various sentry pages still rely on django to serve
-  // html views.
-  appConfig.devServer = !SENTRY_EXPERIMENTAL_SPA
-    ? appConfig.devServer
-    : {
-        ...appConfig.devServer,
-        before: () => undefined,
-        publicPath: '/_assets',
-        proxy: {'/api/': backendAddress},
-        historyApiFallback: {
-          rewrites: [{from: /^\/.*$/, to: '/_assets/index.html'}],
-        },
-      };
```

Comment:
This previously wasn't really used except to test the deploy preview configuration locally, but has been adapted to proxy requests to `sentry.io`